osu.Game/Overlays/BeatmapListing/Panels/BeatmapPanelGrid.cs

```diff
@@ -1,25 +1,25 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using osuTK;
-using osuTK.Graphics;
 using osu.Framework.Allocation;
 using osu.Framework.Extensions.Color4Extensions;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
-using osu.Game.Graphics;
-using osu.Game.Graphics.Sprites;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Graphics.Sprites;
 using osu.Framework.Input.Events;
 using osu.Framework.Localisation;
 using osu.Game.Beatmaps;
 using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
 using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osuTK.Graphics;
 
-namespace osu.Game.Overlays.Direct
+namespace osu.Game.Overlays.BeatmapListing.Panels
 {
-    public class DirectGridPanel : DirectPanel
+    public class BeatmapPanelGrid : BeatmapPanel
```

Comment:
How about `GridBeatmapPanel` or `BeatmapGridPanel` instead? As `Grid` and `List` (below) it makes it sound like this is a container for beatmap panels instead.

Reply:
I guess we can do that. I was suffixing for file grouping and navigation, generally how I prefer things but not that fussed.